### PR TITLE
Pulling in rnode.rb

### PR DESCRIPTION
**Pull request by TomVasile**
## Overview
Provide a brief description of what this PR does, and why it's needed.

Provides homebrew script for install of rnode on Mac.

### Which JIRA issue does this PR relate to? If there is not a JIRA issue addressing this work, please create one now and add the link here.

https://rchain.atlassian.net/browse/CORE-1022

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You assigned one person to review this PR

### If you are not a member of the core development team, please confirm:
- [x] You signed the commit. Merging requires a signature. Please see the [developer wiki](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain) for instructions.

### Notes
Not expecting Travis here. Merging to master since this is the initial commit.

Source: https://github.com/ddayan/homebrew-rchain.git

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ensime.sbt
+TAGS
+node-client/__pycache__
+node-client/ve
+research/sharding/ve
+*.pyc
+integration-testing/.pytest_cache
+integration-testing/.virtualenv
+integration-testing.log
+*.log
```
Comment (adaszko) [inline .gitignore]: Why is this file needed?

Comment (TomVasile) [reply .gitignore]: I just pulled over the gitignore from rchain. It can be removed if not needed.

Comment (adaszko) [reply .gitignore]: I don't think it plays any role so I'd in favor of removing it.
